src/components/CompareResults/CompareResultsTable.tsx

```diff
@@ -26,25 +26,25 @@ function CompareResultsTable(props: CompareResultsProps) {
           <TableContainer component={Paper}>
             <Table
               sx={{ minWidth: 650 }}
-              size="small"
-              aria-label="a dense table"
+              size='small'
+              aria-label='a dense table'
             >
-              <PaginatedCompareResults mode={mode} />
+              <PaginatedCompareResults theme={theme} />
             </Table>
           </TableContainer>
         </>
       )}
       {compareResults.loading && (
-        <Box display="flex" justifyContent="center">
-          <Grid item xs={1} className="compare-results-error">
+        <Box display='flex' justifyContent='center'>
+          <Grid item xs={1} className='compare-results-error'>
             <CircularProgress />
           </Grid>
         </Box>
       )}
       {compareResults.error && (
-        <Box display="flex" justifyContent="center">
-          <Grid item xs={6} className="compare-results-error">
-            <Alert severity="error" sx={{ textAlign: 'center' }}>
+        <Box display='flex' justifyContent='center'>
```

Comment:
I don't know why it's complaining about coverage here so I'd appreciate some eyes on this. Thanks!